src/test/java/org/opensearch/sdk/FailingTransport.java

```diff
@@ -0,0 +1,112 @@
+package org.opensearch.sdk;
+
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
```

Comment:
suggest putting these files requiring the ES license header in their own package so they are easily excluded from our license header checker (similar to how we used to exclude the Netty4 code).

Reply:
As changed the test. This file is not needed anymore :)